quack_app/test/components/navbar_test.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:quack_app/components/navbar.dart';
+import 'package:golden_toolkit/golden_toolkit.dart';
+import 'package:quack_app/constants/constants.dart';
+
+void main() {
+  group("NavBar Test", () {
+    const MaterialApp app = MaterialApp(home: NavBar());
+    testGoldens("settings_active", (WidgetTester tester) async {
```

Comment:
I just am asking for clarification, is this test just checking that the settings page on the app is visible and active? (Same for all the other tests that say blank_active)

Reply:
Yes - essentially, we check that the icon changes to the active icon. Now with David's implemented Nav bar, we also check to see if the view updated (we switched to that tab)